x-pack/plugins/session_view/public/components/session_view_detail_panel/helpers.ts

```diff
@@ -60,10 +61,14 @@ export const getDetailPanelProcess = (process: Process | undefined) => {
   if (!process) {
     return processData;
   }
+  const endProcessList = process.events.filter((items) => items.event.action === 'end');
```

Comment:
might be cleaner to create a new function on ProcessImpl to deal with this logic. e.g
process.getEndTime()

getEndTime() {
  const endEvent = this.findEventByAction(this.events, EventAction.end);
  if (endEvent) {
    return endEvent['@timestamp'];
  }
  
  return null;
}